sparse/etc/acdbdata

```diff
@@ -0,0 +1 @@
+/system/etc/acdbdata
```

Comment:
If I understand right it's firmwares for sound dsp.
Vendor library acdbloader try to load files from /etc